data/pg/common-masters/WageSeekerSkills.json

```diff
@@ -2,35 +2,53 @@
     "tenantId": "pg",
     "moduleName": "common-masters",
     "WageSeekerSkills": [
-        {
-            "name": "Unskilled",
-            "code": "UNSKILLED",
```

Comment:
@Ramkrishna-egov  Add few values present in this file 
https://docs.google.com/spreadsheets/d/12EvDqrFhSEaPA2t0eBcaYjoaTz2k8AbQc1JIS8DOtAs/edit#gid=2107109524